src/services/profile.service.js

```diff
@@ -0,0 +1,123 @@
+const { ApiError } = require('../exceptions/api.error');
+const userService = require('../services/user.service');
+const emailService = require('../services/email.service');
+const bcrypt = require('bcrypt');
+const { checkRequired } = require('../utils/checkRequired');
+const { validatePassword } = require('../utils/validatePassword');
+const { validateEmail } = require('../utils/validateEmail');
+
+const passReset = async (accessToken) => {
+  const user = await userService.getByAccessToken(accessToken);
+
+  if (!user) {
+    throw ApiError.badRequest('bad request', {
```

Comment:
The method 'ApiError.badRequest' should be 'ApiError.BadRequest' to match the static method naming convention in 'ApiError'.